docs/pages/branding/about.tsx

```diff
@@ -19,6 +19,187 @@ import BrandingBeginToday from 'docs/src/modules/branding/BrandingBeginToday';
+          alt=""
+          sx={{
+            width: 207,
+            height: 171,
+            position: 'absolute',
+            right: { xs: 10, md: 0 },
+            bottom: -175,
+          }}
+        />
+        <Typography variant="h2">Team</Typography>
+        <Typography sx={{ mt: 1.5, mb: 7, maxWidth: 700 }}>
+          Material-UI is maintained by a group of invaluable core contributors, with the massive
+          support and involvement of the community.
+        </Typography>
+      </Container>
+    </Box>
+  );
+}
+
+const company = [
+  {
+    name: 'Olivier Tassinari',
+    src: '/static/branding/about/olivier.jpg',
+    title: 'Co-founder',
+    location: 'Paris, France',
+    twitter: 'olivtassinari',
+    github: 'oliviertassinari',
+  },
+  {
+    name: 'Matt Brookes',
+    src: '/static/branding/about/matt.jpg',
+    title: 'Co-founder',
+    location: 'London, UK',
+    twitter: 'randomtechdude',
+    github: 'mbrookes',
+  },
+  {
+    name: 'Sebastian Silbermann',
+    src: '/static/branding/about/sebastian.jpg',
+    title: 'Community focus',
+    location: 'Dresden, Germany',
+    twitter: 'sebsilbermann',
+    github: 'eps1lon',
+  },
+  {
+    name: 'Damien Tassone',
+    src: '/static/branding/about/damien.jpg',
+    title: 'Enterprise focus',
+    location: 'Lodon, UK',
+    twitter: 'madKakoO',
+    github: 'dtassone',
+  },
+  {
+    name: 'Marija Najdova',
+    src: '/static/branding/about/marija.jpg',
+    title: 'Community focus',
+    location: 'Skopje, North Macedonia',
+    twitter: 'marijanajdova',
+    github: 'mnajdova',
+  },
+  {
+    name: 'Danail Hadjiatanasov',
+    src: '/static/branding/about/danail.jpg',
+    title: 'Enterprise focus',
+    location: 'Amsterdam, Netherlands',
+    twitter: 'danail_h',
+    github: 'DanailH',
+  },
+];
+
+function BrandingCompany() {
+  return (
+    <Box sx={{ bgcolor: 'greyEA', pt: 12, pb: 11 }}>
+      <Container sx={{ position: 'relative' }}>
+        <Box
+          component="img"
+          src="/static/branding/block1-white.svg"
+          alt=""
+          sx={{
+            width: 293,
+            height: 120,
+            position: 'absolute',
+            right: { xs: 20, md: 0 },
+            bottom: -130,
+          }}
+        />
+        <Typography variant="h3">
+          <UnderlinedText>Company</UnderlinedText>
+        </Typography>
+        <Typography sx={{ mt: 1.5, mb: 7, maxWidth: 700 }}>
+          The development of the project and its ecosystem is guided by an international team.
+        </Typography>
+        <Grid container spacing={4}>
+          {company.map((member) => (
+            <Grid item xs={12} md={6} lg={4} key={member.name}>
+              <BrandingPersona
+                name={member.name}
+                src={member.src}
+                title={member.title}
+                location={member.location}
+                twitter={member.twitter}
+                github={member.github}
+              />
```

Comment:
```suggestion
              <BrandingPersona {...member} />
```

Just saying we can do it.. :)

Reply:
done